How to fix signature verification errors (#351)
owncloud/docs-client-desktop: modules/ROOT/pages/installing.adoc

image::installing/gnome-shell-extension-appindicator-selector.png[width=600,pdfwidth=60%]
--

==== Native Installation

Linux users must follow the instructions on the {desktop-clients-url}[download] page to add the appropriate repository for their Linux distribution, install the signing key and use their package managers to install the Desktop App. Linux users will also update their Desktop App via package manager. The Desktop App will display a notification when an update is available. Note to see xref:installing-shell-integration-packages[Installing Shell Integration Packages]. Overlay icons and a special context menu for your file browsers need to be installed manually.

You will also find links to source code archives and older versions on the download page.

> On Debian based systems, it may happen after a while that when running `sudo apt update`, a notice about a signature verification error can be returned. This can be solved by envoking following command to update all sinatures:

Reviewer: ```suggestion
On Debian-based systems, it may happen after a while that when running `sudo apt update`, a notice about a signature verification error is returned. This can be solved by invoking the following command to update all signatures:
```

> On Debian-based systems, it may happen after a while that when running `sudo apt update`, a notice about a signature verification error is returned. This can be solved by invoking the following command to update all signatures:

Reviewer: ```suggestion
On Debian-based systems, it may happen after a while that when running `sudo apt update`, a notice about a signature verification error is returned. This can be solved by refreshing the keys. On Debian invoke the following command to update all signatures:

[source,bash]
----
sudo apt-key adv --refresh-keys --keyserver keyserver.debian.org
----
```

> 

Reviewer: ```suggestion
For Ubuntu, use:
```

image::installing/gnome-shell-extension-appindicator-selector.png[width=600,pdfwidth=60%]
--

==== Native Installation

Linux users must follow the instructions on the {desktop-clients-url}[download] page to add the appropriate repository for their Linux distribution, install the signing key and use their package managers to install the Desktop App. Linux users will also update their Desktop App via package manager. The Desktop App will display a notification when an update is available. Note to see xref:installing-shell-integration-packages[Installing Shell Integration Packages]. Overlay icons and a special context menu for your file browsers need to be installed manually.

You will also find links to source code archives and older versions on the download page.

On Debian-based systems, it may happen after a while that when running `sudo apt update`, a notice about a signature verification error is returned. This can be solved by refreshing the keys. On Debian invoke the following command to update all signatures:

[source,bash]
----
sudo apt-key adv --refresh-keys --keyserver keyserver.debian.org
----

> For Ubuntu, use:

Reviewer: ```suggestion

For Ubuntu, use:

```

image::installing/gnome-shell-extension-appindicator-selector.png[width=600,pdfwidth=60%]
--

==== Native Installation

Linux users must follow the instructions on the {desktop-clients-url}[download] page to add the appropriate repository for their Linux distribution, install the signing key and use their package managers to install the Desktop App. Linux users will also update their Desktop App via package manager. The Desktop App will display a notification when an update is available. Note to see xref:installing-shell-integration-packages[Installing Shell Integration Packages]. Overlay icons and a special context menu for your file browsers need to be installed manually.

You will also find links to source code archives and older versions on the download page.

On Debian-based systems, it may happen after a while that when running `sudo apt update`, a notice about a signature verification error is returned. This can be solved by refreshing the keys. On Debian invoke the following command to update all signatures:

[source,bash]
----

> sudo apt-key adv --refresh-keys --keyserver keyserver.debian.org

Reviewer: ```suggestion
sudo apt-key adv --refresh-keys --keyserver keyring.debian.org
```

----

For Ubuntu, use:

[source,bash]
----

> sudo apt-key adv --refresh-keys --keyserver keyserver.ubuntu.com

Reviewer 2: I have had this problem "for a while now":
```
sudo apt update
...
The following signatures were invalid: EXPKEYSIG 0700205DFD41A71A devel OBS Project <devel@s2.owncloud.com>
...
```

I had looked around "the internet" for what to do and did not discover this. The command here worked - fantastic!
